app/TransactionBuilder.php

```diff
@@ -0,0 +1,96 @@
+<?php
+
+namespace App;
+
+use Illuminate\Support\Facades\DB;
+
+class TransactionBuilder
+{
+    /**
+     * @todo could probably use some refactoring - array_map makes the 
+     * removeNullValuesFromArray method necessary.
+     */
+    public function createFromFile(string $filePath): array
+    {
+        $lines = file($filePath, FILE_IGNORE_NEW_LINES);
+        $transactions = array();
+        foreach ($lines as $currentLine) {
+            try {
+                $transaction = $this->extractTransactionFromLine($currentLine);
+                $transactions[$transaction->transaction_hash] = $transaction;
+            } catch (\exception $e) { // @todo custom exception
+                continue;
```

Comment:
I would just leave out the `continue `as it increases cyclomatic complexity unnecessarily